### PR TITLE
Update signal

### PR DESCRIPTION
**Pull request by Loyalsoldier**

### CODE REVIEW

File: data/signal [hunk 1]
```diff
@@ -1 +1,3 @@
+signal.art
 signal.org
+signalusers.org
```
Comment (Robot-DaneelOlivaw) [inline data/signal:3]: Signal Community is an unofficial forum. Is it appropriate to add it here?

**Comment by EpLiar** [comment]
@Robot-DaneelOlivaw https://community.signalusers.org/about

**Comment by Robot-DaneelOlivaw** [comment]
FYI, I found the announcement here: [Welcome to the Community - Announcements - Signal Community](https://community.signalusers.org/t/welcome-to-the-community/8). The real question is should we add domains of unofficial sites(communities, wikis, etc.)?

**Comment by EpLiar** [comment]
I messed up

[This article](https://community.signalusers.org/t/welcome-to-the-community/8) said that this forum is an unofficial site. But [this page](https://community.signalusers.org/about) said if "in the event of a critical issue or urgent matter affecting this site, please contact us at support@signal.org."

**Comment by Loyalsoldier** [comment]
It's at the bottom of signal's landing page.

image

**Comment by Robot-DaneelOlivaw** [comment]
They seem to be pretty close to their community even if it's unofficial. In this case, I guess it's acceptable.

**Pull request merged**